runtime/codert_vm/cnathelp.cpp

```diff
@@ -467,7 +467,7 @@ buildJITResolveFrameForRuntimeCheck(J9VMThread *currentThread)
 	void *oldPC = currentThread->jitReturnAddress;
 #if defined(J9VM_ARCH_X86)
 	U_8 *pc = (U_8*)oldPC;
-	U_32 offset = *(U_32*)pc;
+	I_32 offset = *(U_32*)pc;
```

Comment:
Why was the cast on the right-hand side not also updated?

Reply:
Good point. I will open another PR.